### ISSUE TITLE
Not remembering pin placement after dragging

### ISSUE THREAD

**Issue by paul-frost** [ethercreative/simplemap#20]
I'm getting an issue where if I put in a postcode to get near where I want and then I drag the pin to the exact location (which is still be in the same postcode area), but on saving it reverts to the default postcode lat/lng. Sometimes the postcode area can cover several square kilometres and I need to be able to be precise.
In a particular case there is no second set of numbers for the postcode (Donovan Ln, Taunton TA4, UK) so the area is very large for TA4.
I have 2 separate fields for lat and lng that are using the Preparse plugin to auto populate from the simplemap field data. They store the correct lat/lng from the pin placement after dragging, but on reopening the entry after saving the pin moves back to the default postcode location and then updates the Preparse fields to that.

**Comment by paul-frost** [comment]
I appreciate that this is a free plugin, but it would be good to get an acknowledgement that this is an issue and if it's likely to get fixed or not.
Longer term, is it likely to get update for Craft 3?

I have used this sort of simple Google map field for almost every site I've ever built. Many other plugins offer too many extra functions that I don't need.

**Comment by alexjcollins** [comment]
Hi Paul, firstly, apologies for the somewhat lax approach to maintaining some plugins recently - we've just been super busy on project work. The good news is we're planning on making a bunch of well-needed updates this week so expect something on that soon.

Secondly, we will be making updates to support Craft 3 - just not sure when yet.

Thanks for using and enjoying our plugins!

**Comment by paul-frost** [comment]
Thanks Alex, I switched to using Simplemap when I didn't get any response to bug reports for the paid for plugin I was using. I was getting desperate for a fix, as I have 2 sites close to being finished that are reliant on a Google map field.

All I need is the ability to drag a pin on a map and store the coordinates. I used to use Geotagger on ExpressionEngine sites, and it just wrote/mapped the data direct into existing custom fields. So you created normal custom fields for Town, Postcode, Lat, Long etc. and then in the entry page you entered the town and/or postcode clicked find and a pin appeared, you then dragged the pin to the exact location and the Lat & Long were stored on save. So the Town and Postcode stayed fixed even after you dragged the pin and everything was written into the normal fields rather than a separate table in the database. That way if the plugin didn't work anymore the data was still usable.
That's why I've been using the preparse plugin to save the data from Simplemap into normal fields, as a form of insurance.

I build orienteering club websites and events are usually in a forest nowhere near a house with a postcode, so the usual geolocation search on an address is pretty useless. [Example](http://www.quantockorienteers.co.uk/events/lydeard-hill-and-muchcare-wood-qofl-5-2017-02-19)
